docs: added wg-annoucement draft (#1213)
asyncapi/community: docs/wg.md

# Aloha Maintainers!

Hey beautiful people, 

> I'm thrilled to share some exciting news with you. We've taken a significant leap forward in our journey towards fostering a vibrant and inclusive community by establishing the "Maintainers Growth Working Group." This initiative marks a pivotal moment in our collective effort to not only expand our horizons but also deepen our impact within the AsyncAPI organization. 

Reviewer: I have the feeling this paragragh is not adding so much content. Perhaps using less ornaments and be more precise about what it happened: A new WG has been created.

1. **Expanding Our Horizons**: We aim to significantly increase the number of maintainers across the AsyncAPI organization. This will ensure diverse perspectives and skills that will enrich our community and projects. 
2. **Enhance the Onboarding Experience**: We recognize the importance of a smooth and welcoming onboarding process for new maintainers. We aim to enhance this experience, making it more intuitive, informative, and supportive, ensuring that new members feel valued and empowered from day one. 
3. **Embracing Diversity**: A crucial objective of ours is to increase the number of Non-Postman maintainers. We firmly believe that diversity is not just a buzzword, but a strength that can drive innovation and resilience in our community. We're eager to welcome maintainers from all backgrounds and experiences. 
4. **Building a Support System**: We're fully aware of the challenges and responsibilities that come with being a maintainer. That's why we're not just offering a support system, but we're committed to building a robust one. We'll be there to guide, provide resources, and lend a helping hand whenever needed, ensuring that you feel supported and empowered in your role. 

> 5. **Nurturing Talent**: Lastly, we're passionate about identifying and nurturing potential maintainers among our regular contributors. We see immense value in recognizing and encouraging the talents within our community and providing opportunities for growth and leadership. 

Reviewer: I'm not sure I fully understand the following: 

> we're passionate about identifying

Do we mean we are able to recommend maintainers to projects? If so, once we did in the past but wasn't take positively by some maintainers because we were not considering their opinions (like "do them really want new maintainers?").

Author: No, I'm not talking about recommending maintainers for projects. Instead, I'm referring to nurturing contributors so they are prepared if the opportunity to become a maintainer arises. I hope this clarifies my point, @smoya.

We're excited to announce that our doors are now open to welcoming new maintainers! If your project could use our support, don't hesitate to reach out. As we're a relatively small team at the moment, we operate on a first-come, first-served basis. With one project already in the queue, it's wise not to delay getting in touch. We're keen to get started and help you as fast as possible. Whether you're an existing member of our community or considering joining us, we're eager to connect. Let's explore how we can support your journey as a maintainer. Your passion, skills, and unique perspective are precious to us. Together, we're confident that we can achieve something truly remarkable.

> In conclusion, the "Maintainers Growth Working Group" is more than just a working group; it's a testament to our commitment to fostering a nurturing, inclusive, and dynamic community. We're on this journey together, and I'm incredibly excited about what we can achieve as a team. Let's roll up our sleeves and make a difference, one step at a time. 

Reviewer: I would remove this sentence for the same reason I wrote in https://github.com/asyncapi/community/pull/1213/files#r1613532345

> 

Reviewer: I miss some additional call for WG members. worth mentioning if someone else is interested in our matters, please join our Slack channel and our meetings so they can become members of the WG.

4. **Build a Support System**: Recognizing the challenges and responsibilities inherent in the role of a maintainer, we are dedicated to establishing a comprehensive support system. We aim to offer guidance, resources, and assistance whenever needed, fostering a sense of support and empowerment among maintainers.
5. **Nurture Talent**: We are passionate about identifying and nurturing potential maintainers among our regular contributors. We recognize the value of nurturing talents within our community and providing avenues for growth and leadership.

## Invitation to Collaborate
We are pleased to invite prospective maintainers to join our ranks. If your project stands to benefit from our support, we encourage you to reach out. Our community is currently modest, so we operate on a first-come, first-served basis. With one project already in the pipeline, prompt action is advisable. Whether you are a longstanding member of our community or contemplating joining us, we eagerly anticipate the opportunity to connect. 

## How to contact us

> If you have any questions or are interested in becoming a member of this working group, please contact us by joining our [Slack workspace](https://www.asyncapi.com/slack-invite), and joining the dedicated Working Group channel #wg-maintainers-growth where we discuss all aspects of maintainers' growth. Please ensure you follow our [Slack etiquette](https://github.com/asyncapi/.github/blob/master/slack-etiquette.md) and adhere to the [code of conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
If you have any questions or are interested in becoming a member of this working group, please get in touch with us by joining our Slack channel [#wg-maintainers-growth](https://asyncapi.slack.com/archives/C06GGK7EMSA) in [AsyncAPI's Slack workspace](https://www.asyncapi.com/slack-invite). Please ensure you follow our [Slack etiquette](https://github.com/asyncapi/.github/blob/master/slack-etiquette.md) and adhere to the [Code Of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
```

> You are also welcome to attend any of our bi-weekly meetings by visiting our website at https://www.asyncapi.com/community/events and clicking on any meeting with the title "Maintainers Growth Working Group."

Reviewer: ```suggestion
You are also welcome to attend any of our [live-streamed bi-weekly meetings](https://github.com/asyncapi/community/issues?q=is%3Aissue+Maintainers+label%3Ameeting+%22Maintainers+Growth+Working+Group%22). 
```

5. **Nurture Talent**: We are passionate about identifying and nurturing potential maintainers among our regular contributors. We recognize the value of nurturing talents within our community and providing avenues for growth and leadership.

## Invitation to Collaborate
We are pleased to invite prospective maintainers to join our ranks. If your project stands to benefit from our support, we encourage you to reach out. Our community is currently modest, so we operate on a first-come, first-served basis. With one project already in the pipeline, prompt action is advisable. Whether you are a longstanding member of our community or contemplating joining us, we eagerly anticipate the opportunity to connect. 

## How to contact us

If you have any questions or are interested in becoming a member of this working group, please contact us by:

- Joining our [Slack workspace](https://www.asyncapi.com/slack-invite). Please follow our [Slack etiquette](https://github.com/asyncapi/.github/blob/master/slack-etiquette.md) and adhere to the [code of conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).

> - Joining the dedicated Working Group channel, `#wg-maintainers-growth`, where we discuss all aspects of maintainers' growth. You can find all existing members there.

Reviewer: You added two bullet points as if they were different but both are talking about the same. I would simplify into one and reduce the wording by just mentioning about the channel and a link to the slack invite.

Additionally I miss a sentence explaining we do bi-weekly meetings and how to join.
